FireboltNETSDK/Utils/Constant.cs

```diff
@@ -52,5 +52,7 @@ static class Constant
 
         public static string PROVIDERS_URL = "/compute/v1/providers";
         public static string REGIONS_URL = "/compute/v1/regions";
+
+        public static string ACCOUNT_SYSTEM_ENGINE_URL = "/web/v3/account/{0}/engineUrl";
```

Comment:
C# string format has no easy way to do this except for using third-party libraries. Yet, since we have only one parameter mostly in these strings, we can use `.Replace`

Reply:
Looking further into it, Replace doesn't seem to be the best fit here. I think we should leave ordered parameters here until a better solution is introduced in C#